### PR TITLE
feat: Rewrite fluence-healthcheck

### PR DESCRIPTION
**Pull request by nahsi**

### CODE REVIEW

File: fluence-healthcheck/src/index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  const env = core.getInput("env");
+  const timeout = parseInt(core.getInput("timeout"));
+  let peers, relay;
+
+  const errorMessages = [];
+
+  switch (env) {
+    case "stage":
+      peers = network.stage;
+      relay = network.randomStage();
```
Comment (folex) [inline fluence-healthcheck/src/index.js]: why is it random? we need to check all of them

Comment (folex) [reply fluence-healthcheck/src/index.js]: randomness in healthchecks is usually a pretty bad idea

Comment (nahsi) [reply fluence-healthcheck/src/index.js]: @folex It checks all of them.
This is just a relay to connect to.
Then the script runs connections checks from each peer to all other peers

Comment (folex) [reply fluence-healthcheck/src/index.js]: I understand that. But if a certain relay is down, we will have unpredictable fails.

Comment (nahsi) [reply fluence-healthcheck/src/index.js]: OK I will add a retry to try out a different relay

File: fluence-healthcheck/src/index.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    const data = peers.map(({ peerId, multiaddr }, index) => ({
+      peer: peerId,
+      multiaddr: multiaddr,
+      validators: peers.filter((_, idx) => idx !== index).map((n) => n.peerId),
+    }));
+
+    for (const setup of data) {
+      try {
+        console.log(`Checking peer: ${setup.multiaddr}`);
+        const result = await checkPeer(setup.peer, setup.validators, timeout, {
```
Comment (folex) [inline fluence-healthcheck/src/index.js]: I think it it important to check `result` to contain only `TARGET REACHABLE` and fail on anything else

Comment (folex) [reply fluence-healthcheck/src/index.js]: i don't see how `try catch` will detect problems given that Aqua code has built-in timeout and should always return result

**Pull request merged**